stubs/Pygments/pygments/styles/__init__.pyi

```diff
@@ -6,7 +6,7 @@ from pygments.util import ClassNotFound as ClassNotFound
 
 STYLE_MAP: Mapping[str, str]
 
-def get_style_by_name(name) -> StyleMeta: ...
+def get_style_by_name(name) -> type[StyleMeta]: ...
```

Comment:
Hmm, looks like this should maybe be `type[Style]` rather than `type[StyleMeta]` (You'll have to add `from pygments.style import Style` to the top)

```suggestion
def get_style_by_name(name) -> type[Style]: ...
```

Reply:
That's some neat automation that can catch this! Updated.

Reply:
Thank you! And, yeah, `mypy_primer` is pretty sweet -- you have @hauntsaninja to thank for that!